src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,7 +143,18 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * @param courseId
+     * @param userEmail
+     * @return a list of viewable feedback sessions for any user for his course.
+     * @throws EntityDoesNotExistException
```

Comment:
When I removed it, it did not throw an exception. The subsequent part of the method will simply return an empty list, which is expected because it's querying a non-existent course. I guess the existence check needs to remain there.